daemon/src/maker_cfd.rs

```diff
@@ -67,9 +60,19 @@ pub struct TakerDisconnected {
     pub id: Identity,
 }
 
-pub struct CfdRollOverCompleted {
-    pub order_id: OrderId,
-    pub dlc: Result<Dlc>,
+#[allow(clippy::large_enum_variant)]
+pub enum RollOverCompleted {
```

Comment:
I think for the other protocols, we put this struct into the same file as the actor that produces it. Please do the same here.